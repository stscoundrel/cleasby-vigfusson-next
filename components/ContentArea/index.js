import styles from './ContentArea.module.scss'

export default function ContentArea({ children }) {
  return (
    <section className={styles.section}>
      {children}
    </section>
  )
}
