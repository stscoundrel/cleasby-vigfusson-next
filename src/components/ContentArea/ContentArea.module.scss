.section {
  margin: 30px 0;
  max-width: 750px;
}