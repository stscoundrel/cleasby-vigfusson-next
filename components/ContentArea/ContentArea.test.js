import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import ContentArea from './index'

describe('ContentArea component', () => {
  test('Does not crash', () => {
    const div = document.createElement('div')
    ReactDOM.render(<ContentArea>Content</ContentArea>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('Matches snapshot', () => {
    const tree = renderer.create(<ContentArea>Content</ContentArea>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
