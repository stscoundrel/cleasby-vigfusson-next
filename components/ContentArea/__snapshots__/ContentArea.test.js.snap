// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentArea component Matches snapshot 1`] = `
<section
  className="section"
>
  Content
</section>
`;
